posthog/api/test/test_feature_flag.py

```diff
@@ -348,28 +350,194 @@ def test_updating_a_feature_flag_with_same_team_and_key_of_a_deleted_one(self):
+        feature_flag_instance = FeatureFlag.objects.create(team=self.team, created_by=self.user, key="beta-feature")
+        response = self.client.post(
+            "/api/feature_flag_override",
+            {"feature_flag": feature_flag_instance.id, "user": self.user.id, "override_value": True},
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # String override value
+        feature_flag_instance_2 = FeatureFlag.objects.create(team=self.team, created_by=self.user, key="beta-feature-2")
+        response = self.client.post(
+            "/api/feature_flag_override",
+            {"feature_flag": feature_flag_instance_2.id, "user": self.user.id, "override_value": "hey hey hey"},
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        response = self.client.get("/api/feature_flag_override")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            len(response.json()["results"]), 2,
+        )
+
+    def test_update_override(self):
+        feature_flag_instance = FeatureFlag.objects.create(team=self.team, created_by=self.user, key="beta-feature")
+        response = self.client.post(
+            "/api/feature_flag_override",
+            {"feature_flag": feature_flag_instance.id, "user": self.user.id, "override_value": True},
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        response = self.client.get("/api/feature_flag_override")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            len(response.json()["results"]), 1,
+        )
+        self.assertEqual(
+            response.json()["results"][0]["override_value"], True,
+        )
+
+        response = self.client.post(
+            "/api/feature_flag_override",
+            {"feature_flag": feature_flag_instance.id, "user": self.user.id, "override_value": False},
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        response = self.client.get("/api/feature_flag_override")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            len(response.json()["results"]), 1,
+        )
+        self.assertEqual(
+            response.json()["results"][0]["override_value"], False,
+        )
+
+        feature_flag_override_id = response.json()["results"][0]["id"]
+        response = self.client.patch(
+            f"/api/feature_flag_override/{feature_flag_override_id}", {"override_value": True},
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        response = self.client.get("/api/feature_flag_override")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            response.json()["results"][0]["override_value"], True,
+        )
+
+    def test_create_override_error(self):
```

Comment:
Nit: Clarify test description, what's the expected failure test case? e.g. `test_create_override_with_invalid_value`